lms/djangoapps/static_template_view/tests/test_views.py

```diff
@@ -92,4 +90,3 @@ def test_500(self):
             ),
             status_code=500
         )
-        self.assertContains(resp, settings.TECH_SUPPORT_EMAIL, status_code=500)
```

Comment:
status code check should remain here. We have removed the text only.

Reply:
@DawoudSheraz  on line number `81` and `91`  the response code is already being checked. And it seems  500 check was already redundant.